__tests__/app.test.js

```diff
@@ -24,4 +24,31 @@ describe('GET /api/topics', () => {
     test('404: returns an error for an invalid endpoint', () => {
         return request(app).get('/api/invalid-endpoint').expect(404)
     })
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('200: returns an article object by its ID', () => {
+        return request(app).get('/api/articles/1').expect(200).then(({ body }) => {
+            const { article } = body
+            expect(article.article_id).toEqual(expect.any(Number))
```

Comment:
You need to check the article_id is the one you've requested. This will pass for any article at the moment.